internal/scorer/algorithm/registry.go

```diff
@@ -16,6 +16,8 @@ package algorithm
 
 import "fmt"
 
+// TODO: Replace global variable.
+
```

Comment:
nit: Probably worth removing this comment or extending it to provide some context - I'm not sure what you're thinking here.

Reply:
I have removed it 😃